cmd/thanos/query.go

```diff
@@ -442,11 +441,11 @@ func runQuery(
 				GODEBUG:        os.Getenv("GODEBUG"),
 			}
 		}
-
+		// Use stdout for logging out the request details.
+		logMiddleware := logging.NewServerMiddleware(logger)
```

Comment:
this is for grpc or HTTP? If for one of those, worth to comment (:

Reply:
Done